examples/example_wallet_hwi_signer/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "example_wallet_hwi_signer"
+version = "0.1.0"
+edition = "2021"
+authors.workspace = true
+
+[features]
+simulator = []
+
+[dependencies]
+bdk_wallet = { path = "../../wallet", features = ["file_store"] }
+tokio = { version = "1", features = ["rt", "rt-multi-thread", "macros"] }
+bitbox-api = {version = "0.6.0", features = ["tokio", "simulator"]}
```

Comment:
You probably can have this as an optional dependency with the `simulator` feature.